README.md

```diff
@@ -64,6 +64,41 @@ var ow = openwhisk(options);
 ow.actions.invoke('sample').then(result => console.log(result))
 ```
 
+### Using 3rd party Authentication Handler
```

Comment:
Change this to a `####` header with lower-case title to be consistent with the overall section style?